### ISSUE TITLE
[custom-fields] When adding a new box, all proposal forms give an error

### ISSUE THREAD

**Issue by marta-platoniq** [Platoniq/decidim-module-decidim_awesome#81]
**Describe the bug**
Quan afegeixo una nova caixa "custom-fields" es guarda automàticament sense necessitat d'apretar "Actualitza la configuració". Llavors passa que només clicar en afegir una nova box, com que no hi ha restriccions i es guarda automàticament, em fallen tots els formularis de propostes amb l'error: "Content couldn't be parsed: DL/DD elements not found in the XML".

**To Reproduce**
- decidim-awesome > Proposal Custom Fields > Add a new "custom fields" box
- intentar crear una nova proposta en qualsevol component propostes (primer pas):
<img width="598" alt="image" src="https://user-images.githubusercontent.com/72607737/116382223-f8fc5c80-a815-11eb-93ff-4681efcb1fb9.png">

**Expected behavior**
La configuració no s'hauria de guardar fins que no li doni al botó d'actualitzar.

**Perfeccionismes marta**
Si una box queda buida, hauríem de detectar-ho i no fer-li cas?

**Comment by microstudi** [comment]
bona @marta-platoniq que en penses de inclus ser més restrictius i obligar a posar almenys alguna restricció a aquest qualsevol box? es a dir no permetre que custom fields s'apliquin de manera global en cap cas.

**Comment by microstudi** [comment]
> **Perfeccionismes marta**
> Si una box queda buida, hauríem de detectar-ho i no fer-li cas?

Com a mímin

**Comment by marta-platoniq** [comment]
> bona @marta-platoniq que en penses de inclus ser més restrictius i obligar a posar almenys alguna restricció a aquest qualsevol box? es a dir no permetre que custom fields s'apliquin de manera global en cap cas.

sí, ben pensat té molta lògica això i no té massa sentit que s'apliquin de manera global (que, per altra banda, pot portar a errors/confusions fàcilment). M'encanta la proposta!

**Comment by marta-platoniq** [comment]
Fixed :)

![image](https://user-images.githubusercontent.com/72607737/136197326-bf56fb49-9405-4510-bbb5-34b36eaaccb7.png)
